frontend/public/actions/k8s.ts

```diff
@@ -71,8 +71,11 @@ export const getResources = () => (dispatch: Dispatch) => {
     });
 };
 
-export const filterList = (id: string, name: string, value: string) =>
-  action(ActionType.FilterList, { id, name, value });
+export const filterList = (
+  id: string,
+  name: string,
+  value: string | string[] | { selected: Set<string>; all: string[] },
```

Comment:
```suggestion
  value: string | string[] | { selected: string[]; all: string[] },
```
We should not put non-serializable objects in the redux store.
https://redux.js.org/faq/organizing-state#can-i-put-functions-promises-or-other-non-serializable-items-in-my-store-state

Reply:
not new, but yes, you're right. I've updated the code to not pass `Set` to store anymore